cnf-tests/testsuites/pkg/performanceprofile/performanceprofile.go

```diff
@@ -78,6 +79,50 @@ func FindOrOverridePerformanceProfile(performanceProfileName, machineConfigPoolN
 	return nil
 }
 
+func OverridePerformanceProfile(performanceProfileName, machineConfigPoolName string, newPerformanceProfile *performancev2.PerformanceProfile) error {
+
+	previousPerfProfile, err := FindDefaultPerformanceProfile(performanceProfileName)
+	if err != nil {
+		if !errors.IsNotFound(err) {
+			return err
+		}
+		previousPerfProfile = nil
+	}
+
+	mcp := &mcv1.MachineConfigPool{}
+	err = client.Client.Get(context.TODO(), goclient.ObjectKey{Name: machineConfigPoolName}, mcp)
+	if err != nil {
+		return err
+	}
+
+	if previousPerfProfile != nil {
+		OriginalPerformanceProfile = previousPerfProfile.DeepCopy()
+
+		// Clean and create a new performance profile for the dpdk application
+		err = CleanPerformanceProfiles()
```

Comment:
I think we should not clean wait and then apply and wait again. this means for a multi node cluster that we will wait much more time then needed.

we can just remove and apply in the same time. it will reboot the first node to remove the profile but the other nodes will already apply the new policy on the first reboot.

also another option is to just apply the policy to a new machine config pool and move the nodes to that pull